demo/src/main/java/com/project/demo/entity/Airport.java

```diff
@@ -0,0 +1,49 @@
+package com.project.demo.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Table(name = "VO_Airport")
+@Entity
+public class Airport {
+    @Id
+    private Integer airport_id;
+    private String name;
+    private String id;
```

Comment:
What is the propose of this property?

Reply:
the airport_id is the number identifier and the id is the code or airport that google flights response throws.